packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/redeemable_payments_app_extension_schema.ts

```diff
@@ -0,0 +1,55 @@
+import {BaseSchema} from '../../schemas.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export type RedeemablePaymentsAppExtensionConfigType = zod.infer<typeof RedeemablePaymentsAppExtensionSchema>
+
+const MAX_LABEL_SIZE = 50
+
+export const RedeemablePaymentsAppExtensionSchema = BaseSchema.extend({
+  targeting: zod.array(zod.object({target: zod.literal('payments.redeemable.render')})).length(1),
```

Comment:
Do we have a subset of this properties that will be common to all of the Payments Schemas? We could extract those and extend them to avoid duplication 🤔 
Maybe I'm getting ahead of myself and this will be done at a later step.

Reply:
We might; the idea is to refactor that after all schemas are defined, so we avoid as much toil work as possible 😄